src/WebJobs.Extensions/Extensions/Timers/Scheduling/StorageScheduleMonitor.cs

```diff
@@ -17,19 +18,23 @@ namespace Microsoft.Azure.WebJobs.Extensions.Timers
     public class StorageScheduleMonitor : ScheduleMonitor
     {
         private const string HostContainerName = "azure-webjobs-hosts";
-        private readonly JobHostConfiguration _hostConfig;
+        private readonly JobHostOptions _hostOptions;
+        private readonly DistributedLockManagerContainerProvider _lockContainerProvider;
```

Comment:
We need to review all the changes made during the storage binding split. There are a lot of new public types and surface area that looks questionable to me.

Reply:
Agreed, that is definitely one of them.